rt/emul/mini/src/main/resources/org/apidesign/vm4brwsr/emul/lang/java_lang_Number.js

```diff
@@ -1,572 +1,2655 @@
+  return this
+});
+function $is_jl_Throwable(obj) {
+  return (!(!((obj && obj.$classData) && obj.$classData.ancestors.jl_Throwable)))
+}
+function $isArrayOf_jl_Throwable(obj, depth) {
+  return (!(!(((obj && obj.$classData) && (obj.$classData.arrayDepth === depth)) && obj.$classData.arrayBase.ancestors.jl_Throwable)))
+}
+/** @constructor */
+function $c_s_util_hashing_MurmurHash3$() {
+  this.seqSeed$2 = 0;
+  this.mapSeed$2 = 0;
+  this.setSeed$2 = 0;
+  $n_s_util_hashing_MurmurHash3$ = this;
+  this.seqSeed$2 = $f_T__hashCode__I("Seq");
+  this.mapSeed$2 = $f_T__hashCode__I("Map");
+  this.setSeed$2 = $f_T__hashCode__I("Set")
+}
+$c_s_util_hashing_MurmurHash3$.prototype = new $h_s_util_hashing_MurmurHash3();
+$c_s_util_hashing_MurmurHash3$.prototype.constructor = $c_s_util_hashing_MurmurHash3$;
+/** @constructor */
+function $h_s_util_hashing_MurmurHash3$() {
+  /*<skip>*/
+}
+$h_s_util_hashing_MurmurHash3$.prototype = $c_s_util_hashing_MurmurHash3$.prototype;
+var $d_s_util_hashing_MurmurHash3$ = new $TypeData().initClass({
+  s_util_hashing_MurmurHash3$: 0
+}, false, "scala.util.hashing.MurmurHash3$", {
+  s_util_hashing_MurmurHash3$: 1,
+  s_util_hashing_MurmurHash3: 1,
+  O: 1
+});
+$c_s_util_hashing_MurmurHash3$.prototype.$classData = $d_s_util_hashing_MurmurHash3$;
+var $n_s_util_hashing_MurmurHash3$ = (void 0);
+function $m_s_util_hashing_MurmurHash3$() {
+  if ((!$n_s_util_hashing_MurmurHash3$)) {
+    $n_s_util_hashing_MurmurHash3$ = new $c_s_util_hashing_MurmurHash3$()
+  };
+  return $n_s_util_hashing_MurmurHash3$
+}
+function $f_sr_BoxedUnit__toString__T($thiz) {
+  return "undefined"
+}
+function $f_sr_BoxedUnit__hashCode__I($thiz) {
+  return 0
+}
+var $d_sr_BoxedUnit = new $TypeData().initClass({
+  sr_BoxedUnit: 0
+}, false, "scala.runtime.BoxedUnit", {
+  sr_BoxedUnit: 1,
+  O: 1,
+  Ljava_io_Serializable: 1
+}, (void 0), (void 0), (function(x) {
+  return (x === (void 0))
+}));
+function $f_jl_Boolean__toString__T($thiz) {
+  var b = (!(!$thiz));
+  return ("" + b)
+}
+function $f_jl_Boolean__hashCode__I($thiz) {
+  return ((!(!$thiz)) ? 1231 : 1237)
+}
+var $d_jl_Boolean = new $TypeData().initClass({
+  jl_Boolean: 0
+}, false, "java.lang.Boolean", {
+  jl_Boolean: 1,
+  O: 1,
+  Ljava_io_Serializable: 1,
+  jl_Comparable: 1
+}, (void 0), (void 0), (function(x) {
+  return ((typeof x) === "boolean")
+}));
+function $f_jl_Character__toString__T($thiz) {
+  var c = $uC($thiz);
+  return String.fromCharCode(c)
+}
+function $f_jl_Character__hashCode__I($thiz) {
+  return $uC($thiz)
+}
+var $d_jl_Character = new $TypeData().initClass({
+  jl_Character: 0
+}, false, "java.lang.Character", {
+  jl_Character: 1,
+  O: 1,
+  Ljava_io_Serializable: 1,
+  jl_Comparable: 1
+}, (void 0), (void 0), (function(x) {
+  return $isChar(x)
+}));
+/** @constructor */
+function $c_jl_Exception() {
+  this.s$1 = null;
+  this.e$1 = null;
+  this.stackTraceStateInternal$1 = null;
+  this.stackTrace$1 = null
+}
+$c_jl_Exception.protot
```

Comment:
If there is an easy way to get rid of the `MurmurHash` code, I'd be thankful for using it as well. But yes, I know [JVM is hash oriented system](http://wiki.apidesign.org/wiki/Smalltalk).